### PR TITLE
Add docs for multiple xcode versions installed

### PR DESCRIPTION
**Pull request by rseek**
## Proposed changes

Documentation requested in PR to appium-xcuitest-driver https://github.com/appium/appium-xcuitest-driver/pull/564

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/en/advanced-concepts/multiple-xcode-versions.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ```
+  sudo xcode-select -s /Applications/Xcode7.app/Contents/Developer
+  ```
+2. Run appium (from command line or with GUI).
+  ```
+  appium
+  ```
+
+### Environment variable
+No privileges needed, affects only current shell, so appium should be started
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/multiple-xcode-versions.md]: the current shell

File: docs/en/advanced-concepts/multiple-xcode-versions.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+within that shell.
+
+Assuming you want to choose `/Applications/Xcode9.app`:
+1. Set `DEVELOPER_DIR` environment variable.
+  ```
+  DEVELOPER_DIR=/Applications/Xcode9.app/Contents/Developer
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/multiple-xcode-versions.md]: I think it should be `export ...` if the variable is set separately

**Comment by rseek** [comment]
oh thanks, fixed;
should I squash the commits?

File: docs/en/advanced-concepts/multiple-xcode-versions.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+## Running appium with multiple Xcode versions installed
```
Comment (mykola-mokhnach) [inline docs/en/advanced-concepts/multiple-xcode-versions.md]: I'd also change appium to Appium everywhere

**Review by mykola-mokhnach — APPROVED** [review]

File: docs/en/advanced-concepts/multiple-xcode-versions.md (hunk 3 above)
Comment (rseek) [reply docs/en/advanced-concepts/multiple-xcode-versions.md]: fixed

**Review by imurchie — APPROVED** [review]

**Comment by imurchie** [comment]
I've added this to the table of contents as well.

**Pull request merged**